src/app/container/register-tool/register-tool.component.html

```diff
@@ -57,7 +57,7 @@ <h4>Register Tool</h4>
       </div>
       <div *ngIf="selectedOption?.value === OptionChoice.GitHubApps">
         <p>
-          Navigate to GitHub to install our GitHub app on your repositories/organizations. See our
+          Navigate to GitHub to install our GitHub app in your personal account or organization. See our
```

Comment:
Sorry to be sticky on this one, but I think this is still ambiguous. 
Consider https://theeditorsblog.net/2015/08/08/one-adjective-paired-with-multiple-nouns-a-readers-question/

> A reader recently had a question about adjectives modifying multiple (coordinate) nouns. An example of coordinate nouns in a sentence would be—
> “Remember to wear your red hat and mittens.”
> In a sentence such as this, we recognize that red modifies both hat and mittens. For emphasis, you could repeat the adjective, but you don’t have to.
> “Remember to wear your red hat and red mittens.”

In this case, it sounds like we're saying that both the account and organization are personal when we only mean the account actually. 

I think the advice works

>  If the adjective is used only for the second noun, it clearly doesn’t modify the first noun. 

So it becomes

> Navigate to GitHub to install our GitHub app in a organization or your personal account. See our ...

Reply:
Makes sense, will update